### PR TITLE
Add zetachainAthens3 testnet

### PR DESCRIPTION
**Pull request by SyedAhkam**
## Description

Added a new chain.

Reference: https://www.zetachain.com/docs/reference/api/

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

0x137645BC5f1A8efB2BAB22FAb6829DF8f12847BA

### CODE REVIEW

File: packages/chains/src/zetachainAthens3.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Chain } from './types'
+
+export const zetachainAthens3 = {
```
Comment (jxom) [inline packages/chains/src/zetachainAthens3.ts:3]: Should this be in sync w/ ethereum-chains? https://github.com/ethereum-lists/chains/blob/master/_data/chains/eip155-7001.json

```suggestion
export const zetachainAthensTestnet = {
```

File: packages/chains/src/zetachainAthens3.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  id: 7001,
+  name: 'ZetaChain Athens 3',
```
Comment (jxom) [inline packages/chains/src/zetachainAthens3.ts:5]:
```suggestion
  name: 'ZetaChain Athens Testnet',
```

File: packages/chains/src/zetachainAthens3.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  network: 'zetachain-athens-evm',
```
Comment (jxom) [inline packages/chains/src/zetachainAthens3.ts:6]:
```suggestion
  network: 'zetachain-athens-testnet',
```

File: packages/chains/src/zetachainAthens3.ts (hunk 1 above)
Comment (SyedAhkam) [reply packages/chains/src/zetachainAthens3.ts:3]: surely, I could not locate that file myself hence missed that.

**Pull request merged**